contracts/assemblyscript/nep21-basic/__tests__/main.unit.spec.ts

```diff
@@ -152,6 +154,7 @@ describe('get_balance', () => {
 })
 
 describe('get_allowance', () => {
+  // yeah, this is true for `get_access` in NFT repo as well ¯\_(ツ)_/¯
```

Comment:
wait, do you know the keystrokes for that shrug emoji or ?!

Reply:
I have a macOS shortcut saved; anytime I type "shrug" it gets auto-replaced unless I block it by hitting "escape" 😄